test/NuGet.Core.Tests/NuGet.Common.Test/RuntimeEnvironmentHelperTests.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    public class RuntimeEnvironmentHelperTests
+    {
+        [Fact]
+        public void IsWindows_ShouldReturnConsistentValue()
```

Comment:
I think the tests `IsWindows_ShouldReturnConsistentValue`, `IsMacOSX_ShouldReturnConsistentValue` and `IsMacOSX_ShouldReturnConsistentValue` are not useful. They're basically just assertings that `false == false` or `true == true`. I think all 3 tests should be removed.